app/src/renderer/components/governance/LiProposal.vue

```diff
@@ -45,7 +49,7 @@ export default {
         return {
           button: `vote`,
           message: `Voting for this proposal is open`,
-          color: `blue`
+          color: `green`
```

Comment:
and use orange or similar color here

Reply:
no green is good for open voting! like a traffic light! GO GO GO!